### PR TITLE
Fixes aodn/issues#968

### PR DESCRIPTION
**Pull request by pmbohm**
to be Mafia'd

### CODE REVIEW

**Comment by pmbohm** [comment]
```
java -jar ./mafia/target/mafia-1.0.0.jar \
  -url jdbc:postgresql://127.0.0.1/geonetwork  -user postgres -pass postgres  \
  -all \
  -stdout \
  > IMOS-before.txt


java -jar ./mafia/target/mafia-1.0.0.jar \
  -url jdbc:postgresql://127.0.0.1/geonetwork  -user postgres -pass postgres  \
  -all \
  -transform ./scripts/issues-986.xslt \
  -stdout \
  > IMOS-after.txt
```

File: scripts/issues-986.xslt [hunk 1]
```diff
@@ -52,13 +52,9 @@
...
 
-    <!--Update all IMOS phone numbers-->
-    <xsl:template match="gmd:voice[gco:CharacterString = '61 3 6226 7488']">
-        <gmd:voice>
-        <xsl:element name="gco:CharacterString">
-            <xsl:value-of select='string("61 3 6226 7549")'/>
-        </xsl:element>
-        </gmd:voice>
+    <!-- update phone numbers for imos collection-level records -->
+    <xsl:template match="gmd:CI_ResponsibleParty[gmd:organisationName/gco:CharacterString/text() = 'Integrated Marine Observing System (IMOS)']//gmd:voice/gco:CharacterString">
```
Comment (pmbohm) [inline scripts/issues-986.xslt]: Why we wouldn't want to change all our old numbers to the new one?

**Comment by gsatimos** [comment]
@jonescc would you mind merging this?

File: scripts/issues-986.xslt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          <xsl:apply-templates select="." mode="imosRecord"/>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:apply-templates select="." mode="other"/>
+        </xsl:otherwise>
+      </xsl:choose>
+    </xsl:template>   
+
+    <!-- list of collections to exclude from WMS filter checking -->
+    <xsl:variable name="excludedCollections">
```
Comment (jonescc) [inline scripts/issues-986.xslt:22]: Not used!

**Pull request merged**